### PR TITLE
Change dependabot-core license to MIT

### PR DESCRIPTION
**Pull request by abdulapopoola**
We are relicensing dependabot-core to MIT. This PR also includes several changes to the README and CONTRIBUTING files to reference the new license, and updates to Gemspecs to include the new license.

### CODE REVIEW

**Review by jakecoffman — APPROVED** [review]
😎

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 👋 Want to give us feedback on Dependabot, or contribute to it? That's great - thank you so much!
 
-By submitting a contribution, you agree that contribution is licensed to GitHub under the [MIT-0 license](https://github.com/aws/mit-0).
+By submitting a contribution, you agree that contribution is licensed to GitHub under the [MIT license](LICENSE.txt).
```
Comment (jonjanego) [inline CONTRIBUTING.md:5]:
```suggestion
By submitting a contribution, you agree that contribution is licensed to GitHub under the [MIT license](LICENSE).
```

**Review by jonjanego — CHANGES_REQUESTED** [review]

**Review by jonjanego — APPROVED** [review]

**Review by jurre — APPROVED** [review]

**Review by Nishnha — APPROVED** [review]

**Review by carogalvin — APPROVED** [review]

**Comment by andymckay** [comment]
👍🏾 😎

**Comment by jakecoffman** [comment]
@mattt you might enjoy this PR

**Comment by kurtraschke** [comment]
The README still states "This repo provides the logic necessary for hosting your own standalone Dependabot, as long as you're [not reselling Dependabot to others](https://github.com/dependabot/dependabot-core#license)". Presumably the second clause no longer applies? (The section of the README that the link points at has been deleted...)

**Comment by jonjanego** [comment]
> The README still states "This repo provides the logic necessary for hosting your own standalone Dependabot, as long as you're [not reselling Dependabot to others](https://github.com/dependabot/dependabot-core#license)". Presumably the second clause no longer applies? (The section of the README that the link points at has been deleted...)

Good eyes, thanks @kurtraschke! That was an oversight during our changes.

**Review by baslabofhd777-lab — COMMENTED** [review]
مساعدة في اتمتت العمل

**Pull request merged**